projects/packages/search/src/class-rest-controller.php

```diff
@@ -422,11 +422,7 @@ public function get_local_stats() {
 	 */
 	public function product_pricing() {
 		$tier_pricing = Search_Product::get_pricing_for_ui();
-		// We don't want to show the half finished new pricing, even when it's set to open from the API.
-		if ( ! Helper::is_new_pricing_202208_ready() ) {
-			unset( $tier_pricing['pricing_version'] );
-		}
-		// Unless we are testing.
+		// we can force the plugin to use the new pricing by appending `pricing_version=202208` to URL.
```

Comment:
Just a nitpick to this comment, but forcing a new pricing is done by adding `new_pricing_202208=1` instead of `pricing_version=202208`. Unless there's a separate argument somewhere, but I couldn't get it work with `pricing_version=202208`.

Reply:
ops that was a typo

Reply:
fixed